README.md

```diff
@@ -5,40 +5,158 @@ Please edit this README.md file with your answers to the worksheet questions.
 ## Question 1
 
 ### a
+1a. A B  C  x
+    0 0  0  0
+    0 0  1  0
+    0 1  0  0
+    0 1  1  0
+    1 0  0  0
+    1 0  1  0
+    1 1  0  1
+    1 1  1  0
 
 ### b
+1b. A B  C  x
+    0 0  0  0
+    0 0  1  0
+    0 1  0  0
+    0 1  1  0
+    1 0  0  1
+    1 0  1  1
+    1 1  0  0
+    1 1  1  1
 
 ### c
+1c. A B  C  x
+    0 0  0  0
+    0 0  1  1
+    0 1  0  0
+    0 1  1  0
+    1 0  0  1
+    1 0  1  1
+    1 1  0  0
+    1 1  1  1
 
 ### d
+1d. A B  C  D x
+    0 0  0  1 0
+    0 0  1  1 0
+    0 1  0  1 0
+    0 1  1  1 0
+    1 0  0  1 0
+    1 0  1  1 1
+    1 1  0  1 0
+    1 1  1  1 0
+    0 0  0  0 0
+    0 0  1  0 0
+    0 1  0  0 0
+    0 1  1  0 0
+    1 0  0  0 1
+    1 0  1  0 1
+    1 1  0  0 1
+    1 1  1  0 1
 
 ## Question 2
 
 ### a
+![1a circuit](https://user-images.githubusercontent.com/56162439/67575185-ed693a80-f733-11e9-8314-b2d8450e4f12.png)
+
 
 ### b
+![1b circuit](https://user-images.githubusercontent.com/56162439/67575700-0aead400-f735-11e9-9684-9b72a3890996.png)
 
 ### c
+![1c circuit](https://user-images.githubusercontent.com/56162439/67575718-13430f00-f735-11e9-8191-a32bae8917e5.png)
 
 ### d
+![1d circuit](https://user-images.githubusercontent.com/56162439/67575739-1dfda400-f735-11e9-80a8-ea4df4cea7e2.png)
 
 ## Question 3
 
 ### a
-
+3a. A B x =  A B x
+    0 0 1    0 0 1
+    0 1 0    0 1 0
+    1 0 0    1 0 0
+	  1 1 0    1 1 0
+	  Both tables above show that the identities are the same.
 ### b
+3b. A B x =  A B x
+    0 0 1    0 0 1
+    0 1 1    0 1 1
+    1 0 1    1 0 1
+	  1 1 0    1 1 0
+	Both tables above show that the identities are the same.
+
 
 ### c
+3c. A B C x  =  A B C x
+    0 0 0 0 	  0 0 0 0
+    0 0 1 0     0 0 1 0
+    0 1 0 0     0 1 0 0 
+    0 1 1 0     0 1 1 0
+    1 0 0 0     1 0 0 0
+    1 0 1 1     1 0 1 1
+    1 1 0 1     1 1 0 1
+    1 1 1 1     1 1 1 1
+	Both tables above show that the identities are the same.
 
 ### d
+3d. A B C x  =  A B C x
+    0 0 0 0 	  0 0 0 0
+    0 0 1 0     0 0 1 0
+    0 1 0 0     0 1 0 0 
+    0 1 1 1     0 1 1 1
+    1 0 0 1     1 0 0 1
+    1 0 1 1     1 0 1 1
+    1 1 0 1     1 1 0 1
+    1 1 1 1     1 1 1 1
+	Both tables above show that the identities are the same.
 
 ## Question 4
```

Comment:
You're thinking along the right lines, but some of the boolean expressions don't look quite right -- you seem to be reusing the same symbols for different conditions. It helps to write out exactly what A, B etc correspond to in your reasoning.